content/css/concepts/display/display.md

```diff
@@ -214,3 +214,5 @@ Setting a `div` element to behave like a flex box.
   display: flex;
 }
 ```
+
+![css-display-flex-example](https://raw.githubusercontent.com/Codecademy/docs/main/media/css-display-flex-example.jpg)
```

Comment:
```suggestion
![CSS display flex example](https://raw.githubusercontent.com/Codecademy/docs/main/media/css-display-flex-example.jpg)
```
Inside the square brackets is the alt text for the image. It should be a short phrase relating to the image, like so: